documentation/MeetingMinutes/MOM-2023-09-20.md

```diff
@@ -0,0 +1,123 @@
+  - Issue #45 "Add missing details in authN-authZ doc".
+    - To be further discussed in this call.
+  - Issue #46 "Extend AuthN-AuthZ doc with detailed CIBA flow description".
+    - Agreed to be closed.
+  - Issue #57 "Align securitySchemes and security of CAMARA API specs with IdentityAndConsentManagement WG". Raised by Telefonica.
+    - To be further discussed in this call.
+
+- 1 new PR since last call 06/09
+  - #62 "Create CAMARA-API-AuthN-Inventory.md". Related with issue #57.
+  - #64 "Telefonica Orange table comparison". Related with issue #32.
+
+- 2 Ongoing PRs
+  - #33 "Consent-related use cases". Related with issue #21.
+    - No updates since last call. But it was inadvertently CLOSED by Robert Ludovic (Orange) just before this call. Robert will look into it to reopen it. 
+  - #55 "Include network-based auth info in authN-AuthZ doc". Related with issue #45
+    - To be further discussed in this call.
+  
+
+<br>
+
+## Issues and PRs
+
+### Review PR [#55](https://github.com/camaraproject/IdentityAndConsentManagement/pull/55)
+
+- 
+
+### Review PR [#62](https://github.com/camaraproject/IdentityAndConsentManagement/pull/62) - NEW!
+
+- Intended to create a document to provide an overview of security schemes for CAMARA APIs
+
+<br>
+
+| **Issue** | Owner | Description |
+| ----------| ----- | ----------- |
+| **Closed issues** |  |  |
+| N/A |  |  |
+| **New issues** |  |  |
+| [#59](https://github.com/camaraproject/IdentityAndConsentManagement/issues/59) | Orange | **documentation/CAMARA-API-access-and-user-consent.md diagram update purpose/scope**<br> Documentation/CAMARA-API-access-and-user-consent.md diagrams should be updated for POST/ authorize to use `scope` word instead of `purpose` word. A new PR will be created once we get agreement on issue #32. This new PR will correct the flow accordingly, and include the information about the agreed solution in the relevant doc sections. |
+| [#63](https://github.com/camaraproject/IdentityAndConsentManagement/issues/63) | Deutsche Telekom AG | **Usage of OAUTH2.0 client credentials in SimSwap API**<br> Moved to SIM Swap project as requested by Shilpa (DT). However, it was clarified that this issue actually stems from a discussion in the SIM Swap subproject where it was requested to move the discussion to this WG in order to clarify the agreed guidelines regarding the use of client credentials in CAMARA API access. The issue will be moved back to this WG and it is agreed to close it, as it is already clarified in the issue comments that client credentials do not apply to the SIM Swap API, nor to any other API processing personal user data. Dawid (DT) pointed out that in his opinion the introduction of CIBA into SimSwap looks like a "downgrade" of existing solutions that customers are using. But CIBA flow fits perfectly for the mass market,so it should be fine for CAMARA.<br>Issue will be moved back to this WG and CLOSED according to what is mentioned above.  |
```

Comment:
I'd appreciate having a clear statement from legal/regulatory teams about handling user-related data in Camara APIs, including SimSwap API. Today, it is not clear how a legitimate interest use case has to be handled - should we use opt-out for every scenario or not, e.g. bank that delivers banking services and a bank's legitimate interest is to use a sim swap API for fraud prevention, so bank customer cannot opt-out from this feature.